test/NVQPP/qir_test_cond_for_loop-1.cpp

```diff
@@ -0,0 +1,102 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+// clang-format off
+// RUN: nvq++ --target quantinuum --emulate %s -o %basename_t.x && ./%basename_t.x
```

Comment:
On these tests, shouldn't this be validating the output for the test with `FileCheck` on these `RUN` lines? Also, `main` should be returning `0` on success and non-zero on failure. I think you can remove the `assert`.

Reply:
I was following what was done in testConditionalSample.cpp. I'll rework the tests to remove the assert statements.

Reply:
Updated in d90e40e8